api/base.proto

```diff
@@ -137,6 +137,14 @@ message ApiConfigSource {
 message AggregatedConfigSource {
 }
 
+message SdsSecretConfig {
```

Comment:
The EDS/RDS config protos live in rds.proto/eds.proto. Could we put this in sds.proto for consistency?

Reply:
Per previous comment, which disappeared during rebranding, it's added here, otherwise there is dependency cycle between `sds.proto` and `tls_context.proto`.

Perhaps we should pull `tls_context.proto` into `sds.proto`?

Reply:
Ping. Should I merge `tls_context.proto` into `sds.proto` or are you OK with leaving this in `base.proto`?